mlflow/store/file_store.py

```diff
@@ -138,7 +137,17 @@ def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
             rsl += self._get_active_experiments(full_path=False)
         if view_type == ViewType.DELETED_ONLY or view_type == ViewType.ALL:
             rsl += self._get_deleted_experiments(full_path=False)
-        return [self._get_experiment(exp_id, view_type) for exp_id in rsl]
+        experiments = []
+        for exp_id in rsl:
+            try:
+                # trap and warn known issues, will raise unexpected exceptions to caller
+                experiment = self._get_experiment(exp_id, view_type)
+                experiments.append(experiment)
+            except MissingConfigException as rnfe:
+                # Trap malformed experiments and log warnings.
+                logging.warning("Malformed experiment '%s'. Detailed error %s",
```

Comment:
Let's change to 
```
                logging.warning("Malformed experiment '%s'. Detailed error %s",
                                str(exp_id), str(rnfe), exc_info=True)
```

Reply:
will make this as a following on styling change...